wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -130,12 +130,13 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
 
     private var lastPreBlockStage: PreBlockStage = PreBlockStage.None
 
-    private val analytics: AnalyticsService
+    private val analytics: AnalyticsService by lazy {
+        walletApplication.analyticsService
+    }
```

Comment:
This will avoid problems with Firebase Analytics initialization that we had in explore-dash